libs/standard-tests/langchain_standard_tests/integration_tests/tools.py

```diff
@@ -0,0 +1,43 @@
+from langchain_core.tools import BaseTool
+
+from langchain_standard_tests.unit_tests.tools import ToolsTests
+
+
+class ToolsIntegrationTests(ToolsTests):
+    def test_invoke_matches_output_schema(self, tool: BaseTool) -> None:
+        result = tool.invoke(self.tool_invoke_params_example)
+
+        if tool.response_format == "content":
+            content = result
+        elif tool.response_format == "content_and_artifact":
+            # should be (content, artifact)
+            assert isinstance(result, tuple)
+            assert len(result) == 2
+            content, artifact = result
+
+            assert artifact  # artifact can be anything, but shouldn't be none
+
+        # check content is a valid ToolMessage content
+        assert isinstance(content, (str, list))
```

Comment:
if self.tool_invoke_params_example isn't a ToolCall this isn't guaranteed. should probable wrap tool_invoke_params_example in a ToolCall (if needed) before passing to invoke